Packs/Okta/Integrations/Okta_v2/Okta_v2.py

```diff
@@ -215,6 +217,23 @@ def get_readable_factors(raw_factors):
             factors.append(factor)
         return factors
 
+    @staticmethod
+    def get_rate_limit_context(headers: dict) -> dict:
+        """Takes the response headers and returns the rate limit data to the context
+
+        Args:
+            headers (dict): The response headers
+
+        Returns:
+            dict: The rate limit data to the context
+        """
+        rate_limit_context = {
+            "X-Rate-Limit-Limit": headers["x-rate-limit-limit"],
```

Comment:
think we should either not change it, or Xsoarify it, ie RateLimitReset

I think best just to leave lowercase as it was.